CHANGELOG.md

```diff
@@ -60,9 +60,10 @@ _None_
 * Updated GitHub Actions to use macOS 12.  
   [David Jennes](https://github.com/djbe)
   [#956](https://github.com/SwiftGen/SwiftGen/pull/956)
-* Update dependencies such as SwiftLint (and enable some extra rules).  
+* Update dependencies such as SwiftLint (and enable some extra rules).
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/56171fb00fc4c805290287e9d74e210518b6a24f/CHANGELOG.md#L63"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>